doc/api/debugger.md

```diff
@@ -167,6 +167,10 @@ breakpoint)
 
 ## Advanced Usage
 
+### TCP-based protocol
+
+> Stability: 0 - Deprecated: Use [V8 Inspector Integration][] instead. The debug protocol used by the `--debug` flag was removed from V8.
```

Comment:
I had it on multiple lines at first but line breaks inside of `>` were preserved which looked awkward in the rendered version.